aten/src/ATen/native/cuda/UnaryOpsKernel.cu

```diff
@@ -183,6 +184,17 @@ void clamp_max_kernel_cuda(TensorIterator& iter, Scalar max_value) {
   });
 }
 
+void kaiser_window_kernel_cuda(TensorIterator& iter, int64_t window_length, double beta){
+  AT_DISPATCH_FLOATING_TYPES_AND2(ScalarType::Half, ScalarType::BFloat16, iter.dtype(), "kaiser_window_cuda", [&](){
+    AT_SKIP_BFLOAT16_IF_NOT_ROCM(scalar_t, "kaiser_window_cuda", [&] {
+      auto alpha = (window_length - 1) / 2.0;
```

Comment:
As with the C++ kernel, this will be double and it will make the entire computation occur in double. Is that intended? Seems like it should compute in the type it's given? (Yes, the `i0` function will internally compute in the accumulate type.)